img2dataset/downloader.py

```diff
@@ -56,12 +58,20 @@ def resize_keep_ratio(im, desired_size=256):
     old_size = im.shape[:2] # old_size is in (height, width) format
 
     ratio = float(desired_size)/min(old_size)
-    new_size = tuple([int(x*ratio) for x in old_size])
+    new_size = tuple([int(round(x*ratio)) for x in old_size])
```

Comment:
An image with 412px would be resized to 255 instead of 256 because of rounding error.

Reply:
ok